packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -28,6 +28,11 @@ export interface GridExperimentalFeatures {
    * Will be part of the premium-plan when fully ready.
    */
   preventCommitWhileValidating: boolean;
+  /**
+   * Emits a warning if the cell receives focus without also syncing the focus state.
+   * Only works if NODE_ENV=test.
+   */
+  warnIfFocusStateIsNotSynced: boolean;
```

Comment:
```suggestion
  warnIfFocusStateIsNotSyncedInTest: boolean;
```
Should we be 100% explicit ?

Reply:
I'll leave as it is. In the comment already says the condition where it works.

<img width="500" src="https://user-images.githubusercontent.com/42154031/157443073-ff3a5d2f-3417-47de-a16a-0d7d5cae31df.png" alt="image" style="max-width: 100%;">